### PR TITLE
readthrough of all episodes

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR goes through each episode and makes minor changes/corrections. Types of corrections include:

- Updating links to be correct episode numbering.
- Converting "student" to "learner" for consistency".
- Converting he/she to gender neutral "they" for consistent voice.
- Removing colloquialisms and slang.
- Minor format corrections.

### CODE REVIEW

**Comment by ErinBecker** [comment]
I finished a readthrough of all of the episodes. Could someone please take a look through the changes and merge this PR if ready?

**Comment by ErinBecker** [comment]
Note: The changes to the introductions module were a bit more extensive than the others. For most episodes,I was focusing on formatting, typos, etc (as discussed above), but I did a pruning of the introductions episode to try to cut back the large chunks of text.

File: _episodes/02-practice-learning.md [hunk 1]
```diff
@@ -272,14 +270,14 @@ but each of the other answers provides valuable insight.
 >
 >> ## Solution
 >>
->> *   If the child answers 32, she is throwing away the carry completely.
->> *   If she answers 312, she knows that she can't just discard the carried '1',
+>> *   If the child answers 32, they are throwing away the carry completely.
+>> *   If they answers 312, they know that they can't just discard the carried '1',
```
Comment (karenword) [inline _episodes/02-practice-learning.md]: they answer

File: _episodes/11-practice-teaching.md [hunk 2]
```diff
@@ -64,18 +64,17 @@ a textbook publisher,
...
 
 > ## Reading It Isn't Enough
 > Several research studies (in [2007][fincher-warren], [2012][fincher-stories], and [2015][barker-practice]) have shown
 > that teachers don't adopt instructional practices based on reports about the effectiveness of those practices. 
-> Although some may try these practices in the hope it helps them solve a problem, or because they 
-> are the few that are interested in experimenting with them to improve their teaching, social norms, institutional
-> culture, and lack of time and support prevents many teachers from looking beyond the lesson as it has been taught 'all 
-> these years'. Change in teaching doesn't come about through reading about new teaching practices, but by seeing 
+> Social norms, institutional
+> culture, and lack of time and support prevents many teachers from moving out of
```
Comment (karenword) [inline _episodes/11-practice-teaching.md]: prevent

**Comment by ErinBecker** [comment]
Thanks @karenword. I'll make these corrections and merge.

**Pull request merged**